cfgov/jinja2/v1/_includes/organisms/carousel.html

```diff
@@ -58,8 +58,32 @@ <h2>{{ item.title }}</h2>
 </button>
 {% endmacro %}
 
+{#
+    Since desktop uses a button and mobile uses links,
+    we duplicate the markup here but use a div instead of a button.
+#}
+{% macro render_carousel_thumbnail_mobile( item, is_selected=false ) %}
+<div class="o-carousel_thumbnail{% if is_selected %} o-carousel_thumbnail-selected{%endif%}">
+    <li class="o-carousel_thumbnail-layout">
+        <div class="o-carousel_thumbnail-visual">
+            <a href="{{ item.link.url }}">
+                <img class="o-carousel_thumbnail-img"
+                    src="{{ item.image.thumbnail_url }}"
+                    alt="{{ item.image.alt_text }}">
+            </a>
+        </div>
+        <p class="o-carousel_thumbnail-text">
+            <a href="{{ item.link.url }}">
+                {{ item.title }}
```

Comment:
Per @sonnakim's style guide, there's a separate "thumbnail heading" entry (30 characters maximum) separate from the regular "heading" (45 characters maximum). I realize I need to add that to the stub content. Should I do that here or in a separate PR?

Reply:
Separate PR please